### PR TITLE
add dark mode

### LINKED ISSUE #61: add dark mode to site
After #60 is merged.

### PR DESCRIPTION
**Pull request by vsoch**
The slider I struggled with to get in a good position on the page - it did not go nicely into the sidebar because I could not get the top padding to work. This "on the top" seemed to be the best option for the time being - it is not perfect because when you go to ipad size it overlapst the logo, but someone with more css chops can take a stab at this I hope! I also fixed what I consider a bug that the "releases" box on the front page is slightly larger.

ping @hauten in case you can make the slider in a better spot. I think this would be okay to merge anyway and then work on later.

image

- This will close #61 

Signed-off-by: vsoch <vsoch@users.noreply.github.com>

### CODE REVIEW

**Comment by hauten** [comment]
@vsoch is your fork private? I am unable to commit to this PR after `git remote add vsoch git@github.com:vsoch/flux-framework.github.io.git`. It looks like the remote is added, but I get errors trying to fetch or track to your `add/dark-mode` branch.

**Comment by hauten** [comment]
Things I have in progress or am planning locally:
1. Email & GH icons in nav bar
2. Placement of dark mode toggle
3. Contrast of light mode font and button colors inside the cards (contrast is good in dark mode)

**Comment by vsoch** [comment]
@hauten we just need to add you to have write here!

**Comment by vsoch** [comment]
if you look at the PR title you can see the branch is on the repo here:

image

And if you want to make changes I'd recommend pulling this branch to your fork, and then pushing to your fork.

**Comment by vsoch** [comment]
> Things I have in progress or am planning locally:

And these are great! For this particular PR let's just focus on the second for the button placement - we can work on the other individual things separately (so the PR doesn't get too complex)

File: assets/css/main.css [hunk 1]
```diff
@@ -168,3 +215,82 @@ body div {
...
+  color: var(--font-color) !important;
+  border: 1px solid var(--border-color) !important;
+}
+
+  /*Simple css to style it like a toggle switch*/
+  .theme-switch-wrapper {
+    display: flex;
+    align-items: center;
+    padding-top:20px;
+
```
Comment (hauten) [inline assets/css/main.css:241]: add a line under `.theme-switch-wrapper` for:

padding-left:100px;

File: assets/css/main.css [hunk 2]
```diff
@@ -168,3 +215,82 @@ body div {
...
+.list-group, .list-group-item {
+  background-color: var(--bg-color) !important;
+  color: var(--font-color) !important;
+  border: 1px solid var(--border-color) !important;
+}
+
+  /*Simple css to style it like a toggle switch*/
+  .theme-switch-wrapper {
+    display: flex;
+    align-items: center;
```
Comment (hauten) [inline assets/css/main.css:239]: change `center` to `left` in line 239

**Comment by vsoch** [comment]
@hauten feel free to open a new PR with those commits and the ones here, then I can preview! It’s ok if we wind up closing this one.

**Comment by hauten** [comment]
OK I think this one should be merged, then I'll open another PR

**Comment by vsoch** [comment]
@hauten if we squash and merge here it’s going to make it harder for you because you will have commits not in the history. Are you sure?

**Comment by hauten** [comment]
Yep, I'm going to start over completely w/ tracking to upstream b/c I must have messed it up somewhere.

**Comment by vsoch** [comment]
Okay let’s do it! 🤶

**Pull request merged**